File: fizzbuzz.py
def fizzbuzz():
    """Count from 1 to 20 in fizzbuzz fashion.

    >>> fizzbuzz()
    1
    2
    fizz
    4
    buzz
    fizz
    7
    8
    fizz
    buzz
    11
    fizz
    13
    14
    fizzbuzz
    16
    17
    fizz
    19
    buzz
    """

    i = 1

    while i < 21:

        if i % 3 == 0 and i % 5 == 0:
            print "fizzbuzz"

        elif i % 3 == 0:
            print "fizz"

        elif i % 5 == 0:
            print "buzz"

        else:
            print i
        i += 1

################################################################################
if __name__ == "__main__":
    import doctest

    print
    result = doctest.testmod()
    if not result.failed:
        print "ALL TESTS PASSED. GOOD WORK!"
    print

File: other_code_challenges/fizzBuzz.js
function fizzBuzz() {
    
  for (var i = 0; i <= 20; i++) {
    if (i % 3 === 0 && i % 5 === 0) {
      console.log(i, "fizzBuzz");
    }

    else if (i % 5 === 0) {
      console.log(i, "Buzz");
    }

    else if (i % 3 === 0) {
      console.log(i, "fizz");
    }

    else {
      console.log(i);
    }
  }
}

fizzBuzz();